spec/requests/api/v2/conferences_spec.rb

```diff
@@ -41,15 +41,16 @@
         expect(json.count).to eq(2)
       end
 
-      it "will render upcoming conferences" do
-        upcoming_ruby_conference = create(:conference, :ruby, :upcoming)
+      it "will render only v2 conferences" do
+        ruby_conference = create(:conference, :ruby, start_date: Date.parse("14 Oct 2015"))
+        old_ruby_conference = create(:conference, :ruby, start_date: Date.parse("13 Oct 2015"))
```

Comment:
Useless assignment to variable - `old_ruby_conference`.<br>Line is too long. [95/80]